internal/rbd/replicationcontrollerserver.go

```diff
@@ -614,13 +615,14 @@ func (rs *ReplicationServer) ResyncVolume(ctx context.Context,
 			return nil, status.Error(codes.Internal, err.Error())
 		}
 	}
-
+	// convert the last update time to UTC
```

Comment:
nit: empty lines between sections of a function make it more readable, it was not needed to remove it.

Reply:
Brought it back.